internal/common/common.go

```diff
@@ -241,3 +244,45 @@ func GetHostNTPSources(db *gorm.DB, host *models.Host) (string, error) {
 	}
 	return infraEnv.AdditionalNtpSources, nil
 }
+
+func ParsePemCerts(pemCerts []byte) ([]x509.Certificate, bool) {
+	var certs []x509.Certificate
+	for len(pemCerts) > 0 {
+		var block *pem.Block
+		block, pemCerts = pem.Decode(pemCerts)
+		if block == nil {
+			break
+		}
+		if block.Type != "CERTIFICATE" || len(block.Headers) != 0 {
+			continue
+		}
+		cert, err := x509.ParseCertificate(block.Bytes)
+		if err != nil {
+			return nil, false
+		}
+		certs = append(certs, *cert)
+	}
```

Comment:
Usage of `break` and `continue` statement are confusing, and together it just looks bad.
How about:
```golang
	for len(pemCerts) > 0 {
		var block *pem.Block
		block, pemCerts = pem.Decode(pemCerts)
		if block == nil {
			break
		}
		if block.Type == "CERTIFICATE" && len(block.Headers) == 0 {
				cert, err := x509.ParseCertificate(block.Bytes)
				if err != nil {
					return nil, false
				}
				certs = append(certs, *cert)
				}
				cert, err := x509.ParseCertificate(block.Bytes)
				if err != nil {
					return nil, false
				}
				certs = append(certs, *cert)
		}
	}
```